Modules/Scripted/SampleData/SampleData.py

```diff
@@ -178,24 +185,9 @@ def __init__(self, sampleName=None, sampleDescription=None, uris=None, fileNames
             fileNames = [fileNames]
             nodeNames = [nodeNames]
             loadFiles = [loadFiles]
-            loadFileType = [loadFileType]
+            loadFileTypes = [loadFileTypes]
             checksums = [checksums]
 
-        updatedFileType = []
-        for fileName, nodeName, fileType in zip(fileNames, nodeNames, loadFileType):
-            # If not explicitly specified, attempt to guess fileType
-            if fileType is None:
-                if nodeName is not None:
-                    # TODO: Use method from Slicer IO logic ?
-                    fileType = "VolumeFile"
```

Comment:
What about keeping this logic and instead have something like
```
                    fileType = None  # the fileType will be set automatically based on the registered readers.
```

Reply:
The behavior has not changed, the logic just moved to lower level.